tests/base.py

```diff
@@ -8,6 +8,15 @@
 from datetime import timedelta
 
 from tap_tester.base_suite_tests.base_case import BaseCase
+from tap_tester.jira_client import JiraClient as jira_client
+from tap_tester.jira_client import CONFIGURATION_ENVIRONMENT as jira_config
+
+JIRA_CLIENT = jira_client({ **jira_config })
```

Comment:
Looking at the signature of `jira_client`, I see that we are passing in a dictionary. But why isn't this just `jira_client(jira_config)`? Is the copy necessary?

Reply:
> I'd assume to save the original underlying values from being changed?

I guess that's possible, but the original values are read from environment variables. I don't think I've ever seen us write and update those values in tests

Reply:
Removed